scripts/bootstrap.sh.in

```diff
@@ -111,6 +111,7 @@ configure_salt() {
 accept_keys() {
     echo "Accepting keys"
     crictl exec -i $(crictl ps -q --pod $(crictl pods --name salt-master\* -q)) salt-key -Ay
+    sleep 5
```

Comment:
:cry:

Reply:
Yes I know :disappointed:
btw accept key is not in a state and we need to sleep after this to be able to use it